### ISSUE TITLE
Update apollo client

### ISSUE THREAD

**Issue by schrockn** [dagster-io/dagster#728]
"Warning: apollo update available from 1.6.0 to 2.1.6"

Seems like a good idea to keep up-to-date.

**Comment by bengotow** [comment]
Yeah definitely seems like we don't want to fall behind on this... will bump this up.
